### DISCUSSION TITLE
Webview shows webpage(local html file) slowly first time in coldstartup

### DISCUSSION THREAD

**Discussion by wuyouqiang** [MicrosoftEdge/WebView2Feedback#4032] [Q&A]
i found that the web page shows slowly first time in cold start.( about 7-8s) 

it is basically confirmed that it is caused by file io reading and writing. Especially when running, generate a EWebView folder, see the screenshot below link for details.

https://drive.google.com/file/d/1OrkOiK7YI6NHbwWNxC5VFqXQ91-3TaIX/view?usp=share_link

now Is there any way to speed up the webpage display?

see detail in https://learn.microsoft.com/en-us/answers/questions/1185053/msedgewebview2-show-slowly-first-time-in-cold-star?comment=answer-1182081&page=1#comment-1231686

**Comment by novac42** [comment]
Hi @wuyouqiang may you please provide some more information about this?

	• Framework (WinUI, Win32, WinForms, UWP, WPF, etc)
	• SDK and Runtime version.
	• Operating System

Also if you could provide sample app for us to replicate it would be great.

**Comment by wuyouqiang** [comment]
• Framework : Win32 
• SDK and Runtime version : 
WebView2 SDK: 1.0.1264.42
WIL: 1.0.210803.1 
Runtime: 106.0.1370.52
• Operating System:   Win10. 10.0.19041 N/A Build 19041

**Comment by wuyouqiang** [comment]
use the system Microsoft Edge WebView2 Runtime : 111.0.1661.51

the webpage still show slowly.

**Comment by victorhuangwq** [comment]
Hi, I'm wondering if you are still facing this issue?

**Reply by wuyouqiang** [reply]
it still exist in code start.
because it requires a large number of reading and writing files under the user-data path, including the first shutdown.
We currently treat this as a known issue

**Reply by victorhuangwq** [reply]
@vbryh-msft is there a way to improve the performance of reading and writing files in user data path?
